File: 00-old-explore/02_CounterApp/app/src/main/java/com/java/counterapp/MainActivity.java
package com.java.counterapp;

import static java.lang.Integer.parseInt;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView tvTitle;
    TextView tvNumber;
    Button btnClickMe;

    int counter = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tvTitle = findViewById(R.id.tvTitle);
        tvNumber = findViewById(R.id.tvNumber);
        btnClickMe = findViewById(R.id.buttonClickMe);


        btnClickMe.setOnClickListener(v -> {
            //parseInt(tvNumber.getText().toString()))
            tvNumber.setText(""+increaseCounter());
        });
    }

    public int increaseCounter(){
//        this.counter++;
        return ++counter;
    }
}

File: MyApplication/app/src/main/java/com/java/myapplication/MainActivity.kt
package com.java.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.Menu
import android.view.MenuInflater

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    //Menghubungkan menu ke mainActivity
    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.menu_1, menu)
        return true;
    }
}